libnmstate/netapplier.py

```diff
@@ -314,6 +316,8 @@ def assert_ifaces_state(ifaces_desired_state, ifaces_current_state):
 
         iface_dstate, iface_cstate = _cleanup_iface_ethernet_state_sanitize(
             iface_dstate, iface_cstate)
+        _cleanup_iface_ipv6_link_local_addr(iface_dstate, iface_cstate)
```

Comment:
Better follow Till recommendation as noted in commit f800a0e123a69b10b649f5d624bab79bea2ea0a5 (line 317 above).